sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -439,6 +440,16 @@ class Analyzer(
             case s: Star => s.expand(child, resolver)
             case o => o :: Nil
           })
+        case p: Murmur3Hash if containsStar(p.children) =>
```

Comment:
Actually we do support `*` in all functions, as we expand star in `UnresolvedFunction`. The problem is DataFrame API, it will pass in resolved functions. I think we need an abstraction of function expression, and expand star for them.